src/hooks/useSearchManager.js

```diff
@@ -50,20 +43,24 @@ export const useSearchManager = () => {
   }
 
   const updateSortBy = (newSortOrder) => {
-    if (newSortOrder === sortby[0].value) {
-      delete search.sortby
+    if (newSortOrder === defaultOrdering) {
+      delete search.ordering
```

Comment:
Is there any reason why we need to delete the default ordering from the query params?

Reply:
This will be addressed in PR https://github.com/AlexsLemonade/refinebio-web/pull/433.